go.mod

```diff
@@ -5,7 +5,6 @@ go 1.12
 require (
 	github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc // indirect
 	github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf // indirect
-	github.com/campoy/embedmd v1.0.0 // indirect
```

Comment:
go.mod setup is a bit borked here, this is required for generating the markdown file (go generate)

Reply:
These lines are removed because of the `before.hooks` sections in the `.goreleaser.yml`:

```yaml
before:
  hooks:
    - go mod tidy
    - go generate ./...
```

I dug into this a bit, and I think they're still not needed. You make a good point that `embedmd` is needed for generating documentation, but if you look at the `Makefile`, it actually tries to avoid putting that library in the `go.mod` file:

```makefile
embedmd:
	pushd /tmp && GO111MODULES=on go get github.com/campoy/embedmd && popd
```

As far as I know, using that trick basically avoids putting it into the `go.mod` and `go.sum` files, right?

Reply:
Oh right, I changed that line some time ago. Let's go ahead, it shouldn't break. Fingers crossed